doc/sphinx/practical-tools/coqide.rst

```diff
@@ -5,130 +5,165 @@
+The contents of the right-hand panels are specific to the currently-displayed script.
+The status bar is at the bottom.
 
 Managing files and buffers, basic editing
-----------------------------------------------
-
-In the script window, you may open arbitrarily many buffers to edit.
-The *File* menu allows you to open files or create some, save them,
-print or export them into various formats. Among all these buffers,
-there is always one which is the current *running buffer*, whose name
-is displayed on a background in the *processed* color (green by default), which
-is the one where Coq commands are currently executed.
-
-Buffers may be edited as in any text editor, and classical basic
-editing commands (Copy/Paste, …) are available in the *Edit* menu.
-CoqIDE offers only basic editing commands, so if you need more complex
-editing commands, you may launch your favorite text editor on the
-current buffer, using the *Edit/External Editor* menu.
-
-Interactive navigation into Coq scripts
---------------------------------------------
-
-The running buffer is the one where navigation takes place. The toolbar offers
-five basic commands for this. The first one, represented by a down arrow icon,
-is for going forward executing one command. If that command is successful, the
-part of the script that has been executed is displayed on a background with the
-processed color. If that command fails, the error message is displayed in the
-message window, and the location of the error is emphasized by an underline in
-the error foreground color (red by default).
-
-In the figure :ref:`CoqIDE main screen <coqide_mainscreen>`,
+-----------------------------------------
+
+The *File* menu lets you open files, create new buffers, save buffers to a file,
+and print or export them into various formats.
+
+Text editing provides the basic operations such as copy/cut/paste and replace.
+Most editing operations are shown in the *Edit* menu, which shows the equivalent
+keystrokes for operations.  if you need more complex editor commands, you can
+launch your favorite text editor on the current buffer, using the
+*Edit/External Editor* menu. (When you're done editing you must currently must
+reopen the file to see your changes.)  Also note:
+
+- Undo is also available as Ctrl-Z.  Redo is Ctrl-Shift-Z.
+- Select all is Ctrl-A.
+- Commenting and uncommenting the current line or selected text is available in
+  the *Tools* menu.
+
+Files are automatically saved periodically to a recovery file.  For example,
+`foo.v` is saved to `#foo.v#` every 10 seconds by default.  You can change the
+interval in the *Edit / Preferences / Files* dialog.
+
+In the *View* menu, you can set almost a dozen printing options that
+correspond to options that can appear in the script.  For example, "Display
+notations" on the menu corresponds to the :flag:`Printing Notations` flag.  You
+should use the menu instead of controlling these settings in your script.
```

Comment:
Do we want to link to the Options/Flags section of the refman?

Reply:
``` :flag:`Printing Notations` ``` hyperlinks to that specific flag.  I don't think we can link to the flags options index, and the general explanation of flags and options is too basic.  Unfortunately, the menus don't exactly match the flag names, though they are close.

Reply:
Some of the options set here, set multiple options IIRC. Honestly, I found this entire feature poorly executed, since it also explicitly stops you from setting options within the buffer.

Reply:
Reported as https://github.com/coq/coq/issues/16080